### DISCUSSION TITLE
prisma generates creates a package-lock.json

### DISCUSSION THREAD

**Discussion by reflash** [prisma/prisma#10488] [Q&A] [answered]
Hi!

I've been using prisma for some time. I've recently noticed that when I do prisma generate - it creates a package lock json in my project folder.
The issue is that it's a monorepo and prisma is a dependency in just 1 project, so it seems a bit weird to have a package-lock.json in one of subfolders. More to it, I'm using yarn for dependency management

Is it how everything should work now? Is there any "best-practice" approach to that?

**Comment by aqrln** [comment, marked as answer]
This doesn't sound right and it's definitely a bug. Could you please open an issue and share your directory structure in the monorepo and which directory you are running `prisma generate` from? Ideally, even a Git repository with minimal example that mimics your monorepo layout and triggers this bug. Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
